airflow/gcp/hooks/gcs.py

```diff
@@ -323,6 +323,26 @@ def delete(self, bucket_name, object_name):
 
         self.log.info('Blob %s deleted.', object_name)
 
+    def delete_bucket(self, bucket_name: str):
+        """
+        Delete an bucket object from the Google Cloud Storage
+
+        :param bucket_name: name of the bucket which will be deleted
+        :type bucket_name: str
+        """
+
+        client = storage.Client()
+        bucket = client.bucket(bucket_name)
+
+        if not bucket.exists():
+            self.log.warning("Bucket {} not exists".format(bucket_name))
+            return
+        self.log.info("Bucket name is: {}".format(bucket_name))
+
+        self.log.info("Deleting {}".format(bucket.name))
```

Comment:
Maybe:
```python
self.log.info("Deleting %s bucket", bucket.name")
if not bucket.exists():
             self.log.info("Bucket %s already deleted", bucket_name)
bucket.delete()
         self.log.info("Bucket %s deleted", bucket_name)
```

Also, please to use built-in mechanism of log formatting: `log.info("name %s", name)` instead of `log.info("name {}".format(name))`. I think pylitn should complain on that.

Reply:
The pylint can detect `log.info` but not `self.log.info`. `self.log` comes from LoggingMixin

Reply:
Done